components/network/asterisk/Makefile

```diff
@@ -47,12 +50,17 @@ CONFIGURE_SBINDIR=	/usr/sbin
 CONFIGURE_OPTIONS +=	--sysconfdir=/etc
 CONFIGURE_OPTIONS +=	--localstatedir=/var
 CONFIGURE_OPTIONS +=	--without-oss
-CONFIGURE_OPTIONS +=	--with-pjproject-bundled=no
 CONFIGURE_OPTIONS +=	--with-gsm=internal
 CONFIGURE_OPTIONS +=	--with-postgres=$(PG_HOME)
 
 COMPONENT_INSTALL_TARGETS += samples
 
+build:		$(BUILD_32)
+
+install:	$(INSTALL_32)
+
+test:		$(NO_TESTS)
+
 # Build dependencies
```

Comment:
Please remove the explicit targets. They are superfluous with BUILD_BITS= 32, TEST_TARGET=$(NO_TESTS) and include $(WS_MAKE_RULES)/common.mk